tests/parser/EclipseGridTests.cpp

```diff
@@ -956,6 +956,48 @@ static Opm::Deck createMinpvvEqualsCPDeck() {
     return parser.parseString( deckData) ;
 }
 
+static Opm::Deck createMinpvEqualsMinpvvCPDeck() {
+    const char* deckData =
+        "RUNSPEC\n"
+        "\n"
```

Comment:
Is there a reason for not using raw string literals in the new test here?

Reply:
Nope, just simple copy&paste and my usual laziness (to look up how they and linebreak replacement in emacs works).

Changed,